ch.imatt.ClipQR.yml

```diff
@@ -0,0 +1,32 @@
+app-id: ch.imatt.ClipQR
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+command: clipqr
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
no wayland support?

Reply:
Good question :) I will install a wayland wm and try if that works

Reply:
it's easier if you tell which toolkit it is using.

Reply:
fyne.io for the GUI, wayland suport seems already in good shape: https://github.com/fyne-io/fyne/issues/1851

Reply:
The app runs fine in wayland, but screen scan function doesn't work, I will try to fix this later

Reply:
I took a look at this issue and it won't be easy for me to fix it, I think we can go ahead with x11, wayland support will be for another release